### PR TITLE
[PATCH v4] remove scripts/build-pktio-dpdk

### PR DESCRIPTION
**Pull request by muvarov**
Script was introduced to quick build odp with dpdk pktio
support for dev or testing propose. Now it's more easy to
run docker container with same parameters as CI does.

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>

### CODE REVIEW

**Comment by lumag** [comment]
Reviewed-by: Dmitry Eremin-Solenikov

File: DEPENDENCIES [hunk 1]
```diff
@@ -163,26 +163,42 @@ Prerequisites for building the OpenDataPlane (ODP) API
...
-   for fully accelerated odp dpdk platform.
+3.4.2 Native DPDK install
+   # Debian/Ubuntu starting from 18.04
+   $ sudo apt-get install dpdk-dev
+
+3.4.2 Built DPDK from src
+   git clone --branch=17.11 http://dpdk.org/git/dpdk-stable dpdk
+
+   #Make and edit DPDK configuration
+   make config T=${TARGET} O=${TARGET}
```
Comment (matiaselo) [inline DEPENDENCIES]: `TARGET` should be defined.

File: DEPENDENCIES [hunk 2]
```diff
@@ -163,26 +163,42 @@ Prerequisites for building the OpenDataPlane (ODP) API
...
+   pushd ${TARGET}
 
-3.4.1 Building DPDK and ODP with DPDK pktio support
+   #To use I/O without DPDK supported NIC's enable pcap pmd:
+   sed -ri 's,(CONFIG_RTE_LIBRTE_PMD_PCAP=).*,\1y,' .config
```
Comment (matiaselo) [inline DEPENDENCIES]: `popd` missing.

File: DEPENDENCIES [hunk 3]
```diff
@@ -163,26 +163,42 @@ Prerequisites for building the OpenDataPlane (ODP) API
    activated by setting the environment variable ODP_PKTIO_DISABLE_NETMAP.
 
 3.4 DPDK packet I/O support (optional)
-
```
Comment (matiaselo) [inline DEPENDENCIES]: Better to keep this newline to be more consistent.

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**